package.json

```diff
@@ -78,7 +78,7 @@
     "mocha": "^6.1.3",
     "mocha-phantomjs-core": "^2.1.0",
     "name-suggestion-index": "3.0.6",
-    "npm-run-all": "^4.0.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
Perhaps unintentional? Is this change actually needed by this change? If not, let's omit it.